tables/group.py

```diff
@@ -829,6 +839,9 @@ def __getattr__(self, name):
             return self.__dict__[name]
         return self._f_get_child(name)
 
+    def __item__(self, name):
+        return self._f_get_child(name)
+
```

Comment:
Sorry, probably I'm missing something, but what is for  `__item__` exactly? If I'm not it is not a standard dunder.